examples/process_launch.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import tempfile
+import functools
+import asyncio
+
+from kiwipy import rmq
+import plumpy
+
+
+class DummyProcessWithOutput(plumpy.Process):
+    EXPECTED_OUTPUTS = {'default': 5}
+
+    @classmethod
+    def define(cls, spec):
+        super().define(spec)
+        spec.inputs.dynamic = True
+        spec.outputs.dynamic = True
+        spec.output('default', valid_type=int)
+
+    def run(self):
+        self.out('default', 5)
+
+
+def main():
+    with rmq.RmqThreadCommunicator.connect(connection_params={'url': 'amqp://127.0.0.1'}
```

Comment:
Could this connect call be simplified to:

`kiwipy.connect('amqp://127.0.0.1')`

it just looks a little cleaner and I've changed the `kiwipy` docs to all use that too.